test.jl

```diff
@@ -0,0 +1,45 @@
+using MLJBase
```

Comment:
Please do not put this in the top level, it should go. in `test/` subfolder, and give it an appropriate name (put it with the builtins)